### PR TITLE
LPS-187788 Test Scenarios Coverage | Backend

### PR DESCRIPTION
**Pull request by Mikellorza**
Story: https://liferay.atlassian.net/browse/LPS-184032
Subtask: https://liferay.atlassian.net/browse/LPS-187788

### CODE REVIEW

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/12781

File: modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/EditInfoItemStrutsActionTest.java [hunk 1]
```diff
@@ -227,6 +240,53 @@ public void testEditInfoItemRoundedDoubleTooLong() throws Exception {
...
+				LayoutPageTemplateEntryTypeConstants.TYPE_DISPLAY_PAGE, 0,
+				false, 0, 0, 0, WorkflowConstants.STATUS_APPROVED,
+				ServiceContextTestUtil.getServiceContext(_group.getGroupId()));
+
+		InfoItemFormProvider<?> infoItemFormProvider =
+			_infoItemServiceRegistry.getFirstInfoItemService(
+				InfoItemFormProvider.class, _objectDefinition.getClassName());
+
+		InfoForm infoForm = infoItemFormProvider.getInfoForm(
+			"", _group.getGroupId());
```
Comment (ealonso) [inline modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/EditInfoItemStrutsActionTest.java:262]: Hey @Mikellorza,

Only one small change, Could you please use constants for it? StringPool.BLANK

**Comment by Mikellorza** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/137631

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**